next.config.js

```diff
@@ -32,6 +33,7 @@ module.exports = withImages(withSass({
         };
       }
       return config;
-    }
+    },
+    i18n
```

Comment:
if we run npm run export, the i18n is not available yet in nextjs. but i have a few solutions here to make the localization work.